lib/ex_doc/formatter/html/templates.ex

```diff
@@ -30,6 +30,23 @@ defmodule ExDoc.Formatter.HTML.Templates do
     nil
   end
 
+  @doc """
+  Get the attribute string used to define the spec of the given `node`.
+  """
+  def get_spec_attribute(node)
+
+  def get_spec_attribute(%ExDoc.TypeNode{} = node) do
+    if "opaque" in node.annotations do
+      "@opaque"
+    else
+      "@type"
```

Comment:
`@type` and friends is Elixir specific which means it won't be appropriate for e.g. Erlang. We try to keep language-specific functionality in the corresponding `ExDoc.Language.*` module so let's see if we can make the change there.